File: core/android/app/src/main/java/com/aco/fitar/api/models/LoginModel.kt
package com.aco.fitar.api.models
import kotlinx.serialization.*


import com.aco.fitar.api.ModelBase
import com.aco.fitar.api.ModelResponseBase

@Serializable
class LoginModel(var username:String, var password:String) : ModelBase() {

}

@Serializable
class LoginResponse : ModelResponseBase() {
    public var clientID:Int = 0;
    public var session:String="";
    public var password:String="";

    public var username:String="";
    public var firstName:String="";
    public var lastName:String="";
    public var profilePic:String="";

}

File: core/FitAR/FitAR.API/Models/LoginModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FitAR.Web.API.Models
{

  public class LoginInputModel
  {
    public string username { get; set; }
    public string password { get; set; }
  }

  public class LoginModel : ModelBase
  {
    public int clientID { get; set; }
    public string session { get; set; }

    public string username { get; set; }
    public string firstName { get; set; }
    public string lastName { get; set; }
    public string profilePic { get; set; }

  }
}
